Linux Script/configure-linux.sh

```diff
@@ -215,18 +219,23 @@ removeLogglyConf()
   case "$LINUX_DIST_IN_LOWER_CASE" in
   *"ubuntu"* )
   echo "INFO: Operating system is Ubuntu."
+  PKG_MGR="apt-get"
```

Comment:
@mostlyjason : In the case of Rsyslog version 7, we need to install rsyslog-gnutls package. So I have initialised the package manager variable for each Linux distribution. Please refer: [Line-554](https://github.com/loggly/install-script/pull/85/files#diff-fcb43b5e6da2f59da41dc1549eb0fdf9R554)